### ISSUE TITLE
Split extraction of body parts singular and plural names for better translation

### ISSUE THREAD

**Issue by cainiaowu** [CleverRaven/Cataclysm-DDA#21861]
Currently they are treated as one entry in generated .pot file. 

```
#: lang/json/bodypart_from_json.py src/armor_layers.cpp
msgid "R. Foot"
msgid_plural "Feet"

```

I dont know how to make Transifex accept two different translation, so we might as well split them into two entries.

**Comment by Wuzzy2** [comment]
I agree. Using 2 separate strings makes more sense. The Gettext plural form feature is designed for strings which contain an arbitrary number. In this case, however, it's not needed.

If you're unsure, you can add a translator comment or context to the new strings.

**Comment by Qrox** [comment]
This has been fixed by #33276.
